src/style.css

```diff
@@ -2,0 +1,70 @@
+body {
+  margin-right: 8px;
+  margin-left: 8px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+.input {
+  margin-bottom: 20px;
+}
+
+.input__field {
+  box-sizing: border-box;
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: left center;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-weight: 300;
+  color: #3d4e61;
+  font-family: inherit;
+}
```

Comment:
You are setting the width of the `.input__field` to 100%. This could potentially cause overflow if the parent container's width is fixed. If there's no specific requirement to fix the container size, it's better to let the content size dictate it.